### ISSUE TITLE
markdown link to file highlight fails for paths with spaces

### ISSUE THREAD

**Issue by kappanneo** [microsoft/vscode-markdown-tm-grammar#80]
- VSCode Version: 1.50.1-1
- OS Version: Manjaro 20.1.1 Mikah

Steps to Reproduce:

1. open a new markdown file;
2. write something such as  `[text](<path with spaces>)` *;
3. notice how that is highlight differently compared to other links in the form `[text](path_without_spaces)`.

(*angle brackets should allow for links with spaces)

Similar issue: https://github.com/Microsoft/vscode/issues/24763#issue-221783835
Except there the syntax was not complying with the CommonMark spec, while this should.

Again, links work in the PDF visualization, this is a text highlight issue, not a rendering one.

Does this issue occur when all extensions are disabled?: Yes

**Comment by rahul0705** [comment]
I am unsure exactly how to tackle this one but I think editing https://github.com/microsoft/vscode-markdown-tm-grammar/blob/master/markdown.tmLanguage.base.yaml#L424
```
          (<?)((?<url>(?>[^\s()]+)|\(\g<url>*\))*)(>?)      # The url
```
and changing it to:
```
          (<?)(.*?)(>?)      # The url
```
might fix the issue (and updating the corresponding captures)? If this is sufficient I can open an MR tackling this issue too.

FYI using https://github.com/textmate/markdown.tmbundle/blob/master/Syntaxes/Markdown.tmLanguage#L241 as inspiration

**Comment by mjbvz** [comment]
Fixed by 2f9bda284d391f97982f7932ba49eb89b2ac83b1
